README.rst

```diff
@@ -49,8 +49,19 @@ HistomicsTK can be used in two ways:
   `api-docs <https://digitalslidearchive.github.io/HistomicsTK/api-docs.html>`__
   and `examples <https://digitalslidearchive.github.io/HistomicsTK/examples.html>`__
   for more information.
-
-  This can be installed on Linux via ``pip install histomicstk``.
+  
+  Installation instructions on Linux:
+  
+  *To install HistomicsTK using PyPI*::
+  
+  $ pip install histomicstk
+  
+  *To install HistomicsTK from source*::
+  
+  $ git clone https://github.com/DigitalSlideArchive/HistomicsTK/
+  $ cd HistomicsTK/
+  $ python -m pip install scikit-build ninja cython numpy nimfa
```

Comment:
I think this should be the packages in the setup_requires field of setup.py.  That is, `python -m pip install scikit-build Cython cmake setuptools-scm`.  If we need to explicitly install ninja anything else, they should be added to setup_requires.

Reply:
OK, incorporated changes. It turns out `numpy` needs to be explicitly installed so I updated the instructions accordingly.